### ISSUE TITLE
[Enhancement]: Visual feedback of the series in the ABS app

### ISSUE THREAD

**Issue by Hallo951** [advplyr/audiobookshelf#1735]
### Describe the feature/enhancement

Is it possible to transfer the function of the optical feedback via the heard status of the series to the ABS App?

**Comment by advplyr** [comment]
I'm not sure what you mean

**Comment by advplyr** [comment]
I think you are referring to this https://github.com/advplyr/audiobookshelf-app/issues/410

**Comment by Hallo951** [comment]
Yes, that's actually what I mean. I just forgot that I had already written something about this....

In the android app from ABS, there is currently no visual marker (green line under the audiobook/series) on the series page that indicates whether the series has already been listened to or not. Strangely enough, there is this visual feedback in the android app but on the library page. It would be nice if we could do this for the series in the ABS app, as it is on the server.

The second idea was to extend this function. Series or individual audio books that you have started listening to could be highlighted with a yellow line. This would make it immediately clear to everyone that they had started listening to them....

Translated with www.DeepL.com/Translator (free version)
